strongbox-security-api/src/main/java/org/carlspring/strongbox/security/jaas/Role.java

```diff
@@ -52,6 +54,21 @@ public Role(String name,
         this.description = description;
     }
 
+    @Override
+    public boolean equals(Object o)
+    {
+        if (this == o) return true;
```

Comment:
Could you please always use curly braces around the bodies of `if` statements, even if they're one-liners?